### ISSUE TITLE
Photo Gallery: Accessibility Testing

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1462]
Designs recommendation is to leave photo gallery as-is, but need to evaluate and determine if any accessibility items need to be addressed. 

Please QA test for keyboard accessibility and screen reader accessibility the photo gallery block that can be found on this page: 
https://www.clarkcountynv.gov/government/board_of_county_commissioners/district_a/silverado-ranch-community-center-renderings

**QA Acceptance Criteria:** 
1. Given I am on the Silverado ranch community center renderings page, when I am keyboard navigating, then I am able to tab to and access any photos in the photo gallery, interact with their modal, and close the modal when I am done. 
2. Given I am viewing a photo gallery, when using a screen reader, then all elements of the gallery are announced in logical page order and the content is understandable within the page hierarchy. 

**QA Findings: DEV RESOLUTION NEEDED** 
1. Cannot tab through gallery, user must be able to access all photos in gallery via keyboard navigation and select desired photo in gallery to open modal
2. Screen reader should announce modal content (currently screen reader is stuck in background page)
4. All button/icons inside modal need ARIA labels for screen reader announcement
5. Screen reader should announce 'x' to close modal first and user should be trapped inside modal until they close the modal

**Dev Acceptance Criteria**
1. Given I am viewing a photo gallery, when I am keyboard navigating, then I can access all photos in the gallery and select a photo to open it's modal. 
2. Given I am viewing a photo gallery, when using a screen reader, then the modal content should be announced and all modal icons should have aria labels to understand the purpose of the icon. 
3. Given I am viewing a photo gallery, when using a screen reader or keyboard navigating, then I should be trapped inside the modal until I click 'x' to close the modal. 
4. Given I am viewing a photo gallery, when using a screen reader, then the close 'x' should be announced first so I can exit without having to announce through the content inside the modal first.

**Comment by sarah0arnold** [comment]
Included in release on 8/27/25. Closing ticket.
